README.md

```diff
@@ -188,9 +188,34 @@ docsSearchBar({
 
 ##### `enableDarkMode` <!-- omit in toc -->
 
-Allows you to display the searchbar in dark mode. It is useful if your website has dark mode support and you also want the searchbar to appear in a dark version. 
+Allows you to display the searchbar in dark mode. It is useful if your website has dark mode support and you also want the searchbar to appear in a dark version.
 You can always edit the style of the searchbar to match the style of your website.
 
+Example:
+
+```javascript
+docsSearchBar({
+  ...
+  enableDarkMode: true
+})
+```
+
+Dark mode with `enableDarkMode` set to `true` and system mode set to `dark`:
+![docs-searchbar with dark mode](https://user-images.githubusercontent.com/30866152/121330419-4cce8b00-c916-11eb-8cdf-49a3a640ec43.png)
```

Comment:
It is better to add the image in the current repository, in the [dir named `/assets` ](https://github.com/meilisearch/docs-searchbar.js/tree/main/assets). 

following the example in the readme you can use it like this: 


```md
![docs-searchbar.js example](/assets/docs-searchbar-example.png)
```